src/PostalCodes/Generated/PostalCodeFactory.gen.tt

```diff
@@ -0,0 +1,459 @@
+﻿<#@ template language="C#" hostspecific="true" #>
```

Comment:
We can split this file up into separate templates, but first wanted to get initial feedback. Otherwise no point to lose time on this.